packages/components/addon/components/hds/accordion/item/index.hbs

```diff
@@ -0,0 +1,25 @@
+<Hds::DisclosurePrimitive class={{this.classNames}} @isOpen={{@isOpen}} @toggle={{this.toggle}} ...attributes>
+  <:toggle as |t|>
+    <div class="hds-accordion-item__toggle">
+      <button
+        class="hds-accordion-item__toggle-button"
+        type="button"
+        {{on "click" t.onClickToggle}}
+        aria-label={{this.ariaLabel}}
+        aria-expanded={{if this.isOpen "true" "false"}}
+        aria-controls={{this.contentId}}
+      >
+        <FlightIcon @name="chevron-down" @size="24" @isInlineBlock={{false}} />
+      </button>
+      <div class="hds-accordion-item__toggle-content hds-typography-body-200 hds-foreground-primary">
```

Comment:
In the Toggle, the class should be `display-200-semibold`.

Can we also set the color to `--token-color-foreground-strong` instead of `foreground-primary` since it'll be a heading/title?

Reply:
Thanks I can look at this next.

Reply:
Pushed updates